lib/pxutil.js

```diff
@@ -264,6 +266,29 @@ function getTokenObject(cookie, delimiter = ':') {
     return { key: '_px3', value: cookie };
 }
 
+function getGraphqlData(req) {
+    if (!req.path.includes('graphql')) {
+        return null;
+    }
+
+    const queryArray = req.body['query'] ? req.body['query'].split(/[^A-Za-z0-9_]/) : null;
```

Comment:
why 'query' and not GraphqlOperationType.QUERY ?

Reply:
They are ultimately the same string, but they don't represent the same thing. The body looks like this:

```
{
    "query": "query OperationName {\n    __typename\n}"
}
```

The string `'query'` in this line refers to the key, but the `GraphqlOperationType.QUERY` refers to the first word in the value. If you feel strongly I can change it, but bottom line is that they are different.